src/components/specific/visa/visa-add/VisaAdd.vue

```diff
@@ -105,17 +131,27 @@ export default {
       isClosing.value = null;
     });
 
+    const onClose = event => (isClosing.value = event);
+    const getBack = () => (isSelectingValidator.value = false);
+    const getValidatorList = event => (validatorList.value = event.value);
+
     const submit = async () => {
       const dateConform = isDateConform(dateInput);
 
       if (dateConform) {
         try {
           hasDateError.value = false;
-          await createVisa(
+          const res = await createVisa(
             descInput.value,
             formatDate(dateInput.value),
             props.baseInfo
           );
+          const visaId = res.id;
+
+          await validatorList.value.forEach(
```

Comment:
You can't use `await` on a `forEach` method, it doesn't have any effect.
If what you're trying to achieve is to trigger multiple async calls in parallel and wait for them all to finish before continuing then you should use [Promise.all](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Promise/all).
Let me know if you want some help about it. ;-)